.gitattributes

```diff
@@ -0,0 +1,12 @@
+# Set default behavior to automatically normalize line endings.
+* text=auto
+
+# Force batch scripts to always use CRLF line endings so that if a repo is accessed
+# in Windows via a file share from Linux, the scripts will work.
+*.{cmd,[cC][mM][dD]} text eol=crlf
+*.{bat,[bB][aA][tT]} text eol=crlf
+*.{ics,[iI][cC][sS]} text eol=crlf
```

Comment:
hum, I wonder why we should take such files into consideration?
I'm not an expert, but `ics` files are calendar files, aren't they?

Reply:
This is just an auto-generated template. I can drop that particular one if you prefer.

Reply:
Hi @xSAVIKx 
do you mind to drop it?

Reply:
removed 👍